xrviz/compatibility.py

```diff
@@ -0,0 +1,25 @@
+import logging
```

Comment:
We should have our own logger
```
logger = logging.get_logger('xrviz')
```
and then, below
```
logger.debug(...)
```
so that users can filter only logging statements from this package.